### ISSUE TITLE
Plots does not launch on Gnome 44.1 (when installed via the AUR)

### ISSUE THREAD

**Issue by DarkPhoenix645** [alexhuntley/Plots#127]
Plots does not launch on Gnome 44.1 when installed via the AUR, the following error appears when launched from CLI:

```
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/plots/plots.py", line 145, in do_activate
    self.add_equation(None, record=False)
  File "/usr/lib/python3.11/site-packages/plots/plots.py", line 242, in add_equation
    row = formularow.FormulaBox(self)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/plots/formularow.py", line 256, in __init__
    self.color_picker = colorpicker.PopoverColorPicker()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/plots/colorpicker.py", line 52, in __init__
    self.update_color()
  File "/usr/lib/python3.11/site-packages/plots/colorpicker.py", line 57, in update_color
    self.provider.load_from_data(css.encode())
TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)
```

**Comment by alexhuntley** [comment]
Duplicate of #123, which was fixed in c21bb3cf7b5f369031c5d463de32f6ec05df987f. This fix is contained in version 0.8.5, which is already available on Flathub. Otherwise, please wait for the AUR package to be updated (@ahmubashshir).

**Comment by DarkPhoenix645** [comment]
Thanks!
Sorry I missed #123.
